pkg/lib/bundle/generate.go

```diff
@@ -355,11 +324,6 @@ func GenerateAnnotations(mediaType, manifests, metadata, packageName, channels,
 func GenerateDockerfile(mediaType, manifests, metadata, copyManifestDir, copyMetadataDir, workingDir, packageName, channels, channelDefault string) ([]byte, error) {
 	var fileContent string
 
-	chanDefault, err := ValidateChannelDefault(channels, channelDefault)
```

Comment:
If `channelDefault` is not an empty string, we still want to verify that it exists in `channels`.

Reply:
There are cases when that is totally fine to have default channel not on the list. @shawn-hurley Can you explain this further?

Reply:
Default channel mentioned in the bundle != Default channel on the package